app/lib/cognito.server.ts

```diff
@@ -122,6 +186,13 @@ export async function getGroups() {
   return groups
 }
 
+export async function allGroupNames(): Promise<string[]> {
+  const names = (await getGroups())
+    .map((group) => group.GroupName)
+    .filter((group) => typeof group === 'string') as string[]
+  return names
+}
+
```

Comment:
I think it would be nice to show on the page both the group names _and_ their descriptions. So no need for this specialized function that only returns the names.